src/metabase/db/liquibase/h2.clj

```diff
@@ -11,6 +11,46 @@
 (defn- upcase ^String [s]
   (some-> s u/upper-case-en))
 
+(defn- create-udfs!
```

Comment:
not sure how but should we have tests for this?

Reply:
I just updated this function so it only handles JSONObject. Now `migrate-field-json-unfolding-type-test` has 100% good coverage of JSON_VALUE. Migrations don't change, their tests don't either, and neither should the test coverage.

Reply:
can you add a comment here saying it's tested in that migration?